src/manifest.c

```diff
@@ -233,6 +233,9 @@ static struct manifest *manifest_from_file(int version, char *component, bool he
 		if (strncmp(line, "contentsize:", 12) == 0) {
 			contentsize = strtoull(c, NULL, 10);
 		}
+		if (strncmp(line, "actions:", 8) == 0) {
+			post_update_action = strdup(c);
+		}
```

Comment:
Is this header field only intended for the MoM? If so, can you enforce that here?

Also, for 'update', more than one MoM is parsed, so if both MoMs define "actions:", then the post_update_action value for the current MoM will be overwritten by the value from the latest MoM. Is that behavior acceptable? Or do you need two globals?